### PR TITLE
3.2 Fixes issue #810

### PR DESCRIPTION
**Pull request by paniash**
# Changes made

Fixed the ordering of CZ and CX gates to avoid a global phase of -1 as mentioned in #810 

EDIT: Added `job_monitor` for job status update.

### CODE REVIEW

File: content/ch-algorithms/teleportation.ipynb [hunk 1]
```diff
@@ -22580,1081 +22580,66 @@
...
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": 10,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Exception in Tkinter callback\n",
```
Comment (frankharkins) [inline content/ch-algorithms/teleportation.ipynb:22590]: Is this a bug in qiskit?

Comment (paniash) [reply content/ch-algorithms/teleportation.ipynb:22590]: Apologies for that. It is just the error log when using matplotlib with jupyter. I never specified that the plots be `svg`. Is this alright or shall I execute the code with `svg` plots enabled?

Comment (frankharkins) [reply content/ch-algorithms/teleportation.ipynb:22590]: yes please, svg is preferred as they render better on the site

Comment (paniash) [reply content/ch-algorithms/teleportation.ipynb]: Hey Frank! I ran the cells with svg enabled. Also, I added `job_monitor` for job status updates.

**Review by frankharkins — APPROVED** [review]

File: content/ch-algorithms/teleportation.ipynb (hunk 1 above)
Comment (frankharkins) [reply content/ch-algorithms/teleportation.ipynb]: Thank you!

**Pull request merged**